package.json

```diff
@@ -31,7 +33,9 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "mocha --require babel-core/register --require test/setup.js",
+    "test": "npm run test:react && npm run test:contracts",
+    "test:contracts": "truffle test test/contracts/* --network unexisting",
```

Comment:
I found this weird. If I didn't set network, or set it to some existing network, it would say that the contract couldn't be deployed because of gas.

Reply:
I think that by default the test network will be the `development` network which has a `gasLimit`. Did you try a higher one?

Reply:
Yes. But then I had plenty of problems with deploying the contracts to geth, and I read something about a wrong constructor. I made many changes back and forth, until it started working, so I'm not sure what was wrong.
I reported #7, to investigate this.